commands/scanpullrequest/scanpullrequest.go

```diff
@@ -307,7 +329,9 @@ func deleteExistingPullRequestComment(repository *utils.Repository, client vcscl
 	log.Debug("Looking for an existing Frogbot pull request comment. Deleting it if it exists...")
 	comments, err := utils.GetSortedPullRequestComments(client, repository.RepoOwner, repository.RepoName, int(repository.PullRequestDetails.ID))
 	if err != nil {
-		return err
+		return fmt.Errorf(
+			"failed to get the pull request comments. the following details were used in order to fetch the comments: <%s/%s> pull request id: %d. the error received: %s",
```

Comment:
failed to get the pull request comments. the following details were used in order to fetch the comments
^^
failed to get comments, failed to get comments